manim/scene/vector_space_scene.py

```diff
@@ -1000,17 +1000,21 @@ def get_piece_movement(self, pieces: list | tuple | np.ndarray):
 
         Returns
         -------
-        Animation
+        Animation`
```

Comment:
```suggestion
        Animation
```
Was this just a mistake?

Reply:
yes
i didn't notice it